src/pages/CommitQueue.tsx

```diff
@@ -0,0 +1,81 @@
+import React from "react";
+import styled from "@emotion/styled";
+import { PageWrapper } from "components/styles";
+import { useParams } from "react-router-dom";
+import { Subtitle } from "@leafygreen-ui/typography";
+import Badge from "@leafygreen-ui/badge";
+import { uiColors } from "@leafygreen-ui/palette";
+import { useQuery } from "@apollo/react-hooks";
+import get from "lodash/get";
+import { Skeleton } from "antd";
+
+import { CommitQueueCard } from "./commitqueue/CommitQueueCard";
+import {
+  GET_COMMIT_QUEUE,
+  CommitQueueQuery
+} from "../gql/queries/get-commit-queue";
+
+const { gray } = uiColors;
+
+const Header = styled("div")`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  margin-top: 24px;
+  margin-bottom: 16px;
+`;
+const PageTitle = styled(Subtitle)`
+  margin-right: 16px;
+  font-size: 18px;
+`;
+
+const HR = styled("hr")`
```

Comment:
what is HR?

Reply:
https://www.w3schools.com/tags/tag_hr.asp

Reply:
using it for the horizontal line that seperates the title bar from the the rest of the content